### PR TITLE
Use a yellow favicon in DEV

### PR DESCRIPTION
**Pull request by walaura**
## Why are you doing this?
Looks like a sun!! Dotcom & identity do this too and it's super convenient if you have a thousand tabs 

[**Trello Card**](https://trello.com/c/HHqobaNU/139-yellow-favicon-in-dev)

## Screenshots

screenshot 2019-02-07 at 4 28 18 pm

### CODE REVIEW

File: app/controllers/Application.scala [hunk 1]
```diff
@@ -33,12 +33,13 @@ class Application(
     stringsConfig: StringsConfig,
     settingsProvider: AllSettingsProvider,
     guardianDomain: GuardianDomain,
-    stage: Stage
+    val stage: Stage
```
Comment (rupertbates) [inline app/controllers/Application.scala]: Is this needed?

File: app/controllers/PayPalOneOff.scala [hunk 2]
```diff
@@ -27,12 +28,14 @@ class PayPalOneOff(
     paymentAPIService: PaymentAPIService,
     identityService: IdentityService,
     settingsProvider: AllSettingsProvider,
-    tipMonitoring: Tip
+    tipMonitoring: Tip,
+  stage: Stage
```
Comment (johnduffell) [inline app/controllers/PayPalOneOff.scala]: is this indentation alright? looks underindented on github diff

Comment (walaura) [reply app/controllers/PayPalOneOff.scala]: i think my intellij messed up spaces/tabs, will go over today

**Comment by johnduffell** [comment]
> Seems a lot of code to change a favicon doesn't it?! 😃

agree!... but not a lot of code to change the content of every single page?

although on that topic @walaura another option is to keep it in the standard place, link it to a "real" route rather than an assets path.  then you can put the if statement in the FavIconController instead of every single template.

**Comment by walaura** [comment]
> > Seems a lot of code to change a favicon doesn't it?! 😃
> 
> agree!... but not a lot of code to change the content of every single page?
> 
> although on that topic @walaura another option is to keep it in the standard place, link it to a "real" route rather than an assets path. then you can put the if statement in the FavIconController instead of every single template.

Great idea! i actually tried replacing it via webpack too, this is on that line. I'm not sure if the overhead in production is worth it?

**Comment by johnduffell** [comment]
not sure if the favicon is critical for PROD... and anyway we can put super long cache headers in so fastly will basically keep it for ever, so it would prbably be just as quick if not quicker for the one person who hits it first during/after a deploy!

However I don't want to rain on your great idea and it's not a blocker, I just got thinking about why rupert's comment was true!

**Comment by walaura** [comment]
@johnduffell nono, I agree it's A TON of code plus a new extra global peppered all over the place. This is already such a tough merge that it's gonna take work no matter what I do, I'm happy to try to make the route later today!

**Pull request closed without merging**